services/kafka-operator/0.20.0/defaults/cm.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kafka-operator-0.20.0-d2iq-defaults
+  namespace: ${releaseNamespace}
+data:
+  values.yaml: |
+    ---
+    crd:
+      enabled: true
```

Comment:
add newlines to this and the other files

Reply:
fixed in b9c7e9db825e1a20981c34168562adca6d31de14 😅